static/app/views/issueDetails/groupEventCarousel.tsx

```diff
@@ -427,10 +419,6 @@ export function GroupEventCarousel({event, group, projectSlug}: GroupEventCarous
             )}
           </EventIdAndTimeContainer>
         </EventHeading>
-        {/* Once trace-related issues are GA, we will remove this */}
```

Comment:
Somehow, in #73120 I remove `TraceLink` from `groupEventHeader.tsx` instead of this file.